entries/accordion.xml

```diff
@@ -86,6 +86,21 @@
 				</desc>
 			</type>
 		</option>
+		<option name="classes" type="Object" default='Object' example-value='{ "ui-accordion-header": null }'>
```

Comment:
I would change `classes` in the description of the 2nd and 3rd examples to specify that your specifically setting the value for the `ui-accordion-header` property on the classes option